source/d3d8to9_device.cpp

```diff
@@ -705,6 +722,14 @@ HRESULT STDMETHODCALLTYPE Direct3DDevice8::MultiplyTransform(D3DTRANSFORMSTATETY
 }
 HRESULT STDMETHODCALLTYPE Direct3DDevice8::SetViewport(const D3DVIEWPORT8 *pViewport)
 {
+	if (ViewpointFlag && NewScene == false)
```

Comment:
I'm not a fan of the `== false` syntax. Would you mind changing this to `!NewScene`?